doc/rfd/rfd/0009/README.md

```diff
@@ -0,0 +1,256 @@
+---
+authors: Victor Cuadrado <vcuadradojuan@suse.com>, Ettore Di Giacinto <edigiacinto@suse.de>
+state: discussion
+discussion: https://github.com/cloudfoundry-incubator/kubecf/pull/1495
+---
+
+# RFD 9: Specification for deployment and automation scripts
+
+## Context
+
+
+Developing a handful of loosely dependent Helm charts and building a Helm
+distribution from them without creating an all-or-nothing "super-chart" is
+difficult. One needs to make sure that deployment and automation scripts are
+interoperable between charts, repos, repos' histories, and that they can be
+reused and extended downstream as needed.
+
+At the same time one needs to create, reuse, and extend test environments across
+charts, and between upstream and downstream.
+
+The following spec is an interface specification for sharing deployment and
+automation scripts of Helm charts and utility applications.
+
+### User stories
+
+#### Direct downstream developer
+
+As a downstream developer of a project that directly depends on kubecf helm
+chart, I need to have an up-to-date way of deploying, testing, and automating my
+modifications, without needing to pursue upstream and keep daily track of
+changes in upstream automation.
+
+If kubcf chart implements this spec, I will have those tagged together with their
+release.
+I will reuse automation scripts such as test runs, airgap configurations,
+private registry settings, etc.
+Also it will be possible for me to inject workarounds that I may
+upstream later.
+
+My project also depends on minibroker chart, quarks chart, and in the future on helm
+caps-registry chart. I, as downstream, need to come with a way for to test and
+automate them together. The more charts that implement the spec, the more I can
+rely on their example-values and automation.
+
+
+#### Parallel upstream developer
+
+I am a developer of kubecf chart. My chart doesn't directly depend on stratos
+chart or caps-registry chart. But sometimes they develop automation that I could reuse.
+If they implement this spec, I will have access to their automation in simple,
+pull-way manner.
+
```

Comment:
Maybe we should also add CAP / CAP CI user story, here.

Reply:
The "downstream" developer user story is written with that intent. Would that be fitting?